next/next.config.js

```diff
@@ -62,6 +62,19 @@ const nextConfig = {
   },
   async redirects() {
     return [
+      {
+        source: '/o-nas/dokumenty-a-zverejnovanie-informacii/:categorySlug/:slug',
+        destination: '/o-nas/dokumenty-a-zverejnovanie-informacii/:slug',
```

Comment:
this redirect did not seem to work for me for some reason ?

https://user-images.githubusercontent.com/2327312/209363060-5a845286-46ea-43a0-8a5f-5a26c0178960.mov

was running this against staging instance of strapi and meilisearch (copied the whole staging env as my .env.development.local), so just wanted to recheck if that is the reason why I'm still getting the `undefined`s in the result

Reply:
Thanks, there was a mistake in redirects due to disabled locale detection. Adding '/sk/` prefix to source url fixed it.
You also found another bug with undefined category in reach results - fixed and removed too.